e2e/lvm/lvm_suite_test.go

```diff
@@ -33,3 +38,61 @@ var _ = Describe("Validation test", func() {
 		})
 	})
 })
+
+// Test to verify the PVC status
+var _ = Describe("PVC Status check", VerifyPVCStatus)
+
+func VerifyPVCStatus() {
+
+	Describe("pvc", func() {
+		var filepvc *k8sv1.PersistentVolumeClaim
+		var filepod *k8sv1.Pod
+		var blockpvc *k8sv1.PersistentVolumeClaim
+		var blockpod *k8sv1.Pod
+		var namespace string
+
+		BeforeEach(func() {
+			namespace = tests.TestNamespace
+			filepvc = tests.GetSamplePVC(tests.StorageClass, "5Gi", "lvmfilepvc", v1.PersistentVolumeFilesystem)
+			filepod = tests.GetSamplePod("lvmfilepod", "lvmfilepvc")
+			blockpvc = tests.GetSamplePVC(tests.StorageClass, "5Gi", "lvmblockpvc", v1.PersistentVolumeBlock)
+			blockpod = tests.GetSamplePod("lvmblockpod", "lvmblockpvc")
+		})
+
+		AfterEach(func() {
+			err := tests.DeployManagerObj.GetK8sClient().CoreV1().Pods(namespace).Delete(context.TODO(), filepod.Name, metav1.DeleteOptions{})
+			if err != nil && !errors.IsNotFound(err) {
+				Expect(err).To(BeNil())
+			}
+			err = tests.DeployManagerObj.GetK8sClient().CoreV1().PersistentVolumeClaims(namespace).Delete(context.TODO(), filepvc.Name, metav1.DeleteOptions{})
+			if err != nil && !errors.IsNotFound(err) {
+				Expect(err).To(BeNil())
+			}
```

Comment:
PVC deletion will not ensure PV is deleted. you might need to add a check to validate that PV is also deleted.

Reply:
I think we can take that up in a later PR. For now,  the LVMCluster cleanup will fail if the PV still exists.